.github/workflows/coding-standards.yml

```diff
@@ -54,5 +54,5 @@ jobs:
       # https://github.com/doctrine/.github/issues/3
       - name: "Run PHP_CodeSniffer"
         run: |
-          vendor/bin/phpcs -q --report=diff || true
+          vendor/bin/phpcs -q --report-emacs --report-diff || true
```

Comment:
Since the goal is that humans read this, why pick `emacs` instead of `full`?

Reply:
Because `emacs` is perfectly readable and with more errors better for humans, simple is better IMO.

Reply:
>  with more errors better for humans

What is this supposed to mean? `full` or `code` also include non-autofixable errors.

Reply:
Reworded.

Reply:
I think [`--report-code`](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Reporting#printing-a-code-report) would be more consistent with `--report-diff` (it looks a bit similar since you get context lines).

Reply:
I do not like it because file path is not repeated, thus it takes user much more time to find relevant line.

Reply:
Since you're the one using this, let's do as you prefer.